test/e2e/accounts/snap-account-contract-interaction.spec.ts

```diff
@@ -14,8 +14,10 @@ import {
   ACCOUNT_2,
 } from '../helpers';
 import FixtureBuilder from '../fixture-builder';
+import { installSnapSimpleKeyring } from '../page-objects/flows/snap-simple-keyring.flow';
+import { loginWithBalanceValidation } from '../page-objects/flows/login.flow';
 import { SMART_CONTRACTS } from '../seeder/smart-contracts';
-import { installSnapSimpleKeyring, importKeyAndSwitch } from './common';
+import { importKeyAndSwitch } from './common';
```

Comment:
Intended change to use `installSnapSimpleKeyring` flow created following page object model.